src/components/InputField.js

```diff
@@ -1,4 +1,4 @@
-import * as s from "./InputField.sc";
+import * as s from "../pages/info_page_shared/InputField.sc";
```

Comment:
We usually place the styling together with the file - so maybe just move the file also to the components?

Reply:
Done ✅